paimon-spark/paimon-spark-common/src/test/scala/org/apache/paimon/spark/sql/DDLTestBase.scala

```diff
@@ -456,4 +456,12 @@ abstract class DDLTestBase extends PaimonSparkTestBase {
     }.getMessage
     assert(error.contains("Unsupported partition transform"))
   }
+
+  test("Fix caching catalog case sensitive") {
```

Comment:
I ran this test in the master branch, it passed.

Reply:
oh, sorry for the missleading. It seems the cache future is disabled in tests.. I will modify the test.